specs/electra/beacon-chain.md

```diff
@@ -683,9 +683,8 @@ def initiate_validator_exit(state: BeaconState, index: ValidatorIndex) -> None:
 ```python
 def switch_to_compounding_validator(state: BeaconState, index: ValidatorIndex) -> None:
     validator = state.validators[index]
-    if has_eth1_withdrawal_credential(validator):
```

Comment:
We actually do need this check. Arguably just on queue_excess_active_balance, but if w're already a compounding validator and get consolidation into us, then we'd be re-queueing balance we've already waited for if we don't do this check...

The scenario would be pretty normal, for example if Im rolling a bunch of validators together. the first one would happen normally, but the second would queue 32 eth as excess balance. the third would queue 64 eth as excess balance... etc...